presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -335,6 +338,21 @@ protected Boolean visitFunctionCall(FunctionCall node, Void context)
                             PERFORMANCE_WARNING,
                             "COUNT(DISTINCT xxx) can be a very expensive operation when the cardinality is high for xxx. In most scenarios, using approx_distinct instead would be enough"));
                 }
+                if (functionResolution.isApproxDistinctFunction(analysis.getFunctionHandle(node))) {
+                    double maxStandardError = LOWEST_APPROX_DISTINCT_MAX_STANDARD_ERROR;
```

Comment:
This should be in the config/session properties.

Reply:
Could you provide more context on this? (classes, modules etc.)

Reply:
`FeatureConfigs` and `SystemSessionProperties`. There are a lot of examples in those two classes. Feel free to add one more flag

Reply:
FeaturesConfig/SystemSessionProperties classes. See the pattern we use there

Reply:
Thanks for the pointers!

Reply:
@kaikalur @highker What configuration we need here? We can rely on the DefaultApproximateCountDistinctAggregation.DEFAULT_STANDARD_ERROR right?

Reply:
Generally we want these things to be configurable so deployments (or individual queries) can use different values